tool-populate/api/Submissions.js

```diff
@@ -0,0 +1,98 @@
+var _            = require('lodash')
+var Parse        = require('parse/node')
+var DemoData     = require('../data/DemoData')
+var Users        = require('./Users')
+var Helpers      = require('./Helpers')
+var Forms        = require('./Forms')
+var Questions    = require('./Questions')
+var Chance       = require('chance')
+
+var chance       = new Chance()
+var Submission   = Parse.Object.extend("Submission")
+var useMasterKey = {useMasterKey: true}
+
+function loadSubmissions(options, callback) {
+  Helpers.fetchObjects(Submission, callback)
+}
+
+function randomNum(min, max){
+  return chance.integer({min:min,max:max})
+}
+
+function createAnswer(question) {
+  var question = question.toJSON()
+  var props = question.properties
+  var choices = props.choices
+  var answer = null
+  switch(question.type) {
+    case 'multipleChoice':
+      answer = choices[randomNum(0, choices.length - 1)]
+      break
+    case 'checkboxes':
+      var numberChosen = randomNum(0, choices.length),
+          _choices = _.shuffle(choices)
+      answer = []
+      for(var i = 0; i < numberChosen; i++){
+        answer.push(_choices[i])
+      }
+      break
+    case 'longAnswer':
+      answer = chance.paragraph({sentences: randomNum(5, 10)})
+      break
+    case 'shortAnswer':
+      answer = chance.sentence({words: randomNum(1, 15)})
+      break
+    case 'date':
+      // Randomly add repetitive dates
+      random = randomNum(0,2)
+      if (random === 1) answer = new Date('02-14-2016')
+      else answer = chance.date()
+      break
+    case 'datetime':
+      answer = chance.date()
+      break
+    case 'number':
+    case 'scale':
+      answer = randomNum(props.min, props.max)
+      break
+  }
+  return answer
+}
+
+function createSubmission(answers, form, user) {
```

Comment:
A submission needs a uniqueId specified.  It is an md5 hash of user.id and form.id.

See:
https://github.com/ecohealthalliance/GoodQuestion/blob/master/js/api/Submissions.js#L39-L49